sandbox/Generated/UnitGenerator/UnitGenerator.SourceGenerator/FileGenerate.A.Generated.cs

```diff
@@ -1,13 +1,8 @@
-﻿// <auto-generated>
-// THIS (.cs) FILE IS GENERATED BY UnitGenerator. DO NOT CHANGE IT.
-// </auto-generated>
-#pragma warning disable CS8669
-using System;
-
-namespace FileGenerate
+﻿#pragma warning disable CS8669
+using System;namespace FileGenerate
 {
     [System.ComponentModel.TypeConverter(typeof(ATypeConverter))]
-    public readonly partial struct A : IEquatable<A> 
+    readonly partial struct A : IEquatable<A>
```

Comment:
public?

Reply:
The master code seems to omit public as well.
https://github.com/Cysharp/UnitGenerator/blob/master/src/UnitGenerator/CodeTemplate.tt#L34

Probably, since it is partial, the handwritten code will be followed ?

Reply:
yeah, omit modifier is ok.